Makefile

```diff
@@ -1,18 +1,38 @@
 SHELL=/bin/bash
 
-all: clean pypy35.zip upload publish
+PYPY_VERSIONS := pypy-7.1.1 pypy3.5-6.0.0
```

Comment:
This is following the convention of `squeaky/protable-pypy`, right?

Reply:
Indeed.